### ISSUE TITLE
The Mirrazz Cookie !!

### ISSUE THREAD

**Issue by kelbazz** [themirrazz/oops#1]
Hey @themirrazz it's been a while! What's up? Do you have a new gh and discord account?

**Comment by themirrazz** [comment]
hi uh i have an alt as for discord not yet but i can email you from my new email btw I was wondering if it would be okay if I created new repo(s) on Onofficiel for Border 1.11,Desktop, Android, and iOS, and for BRX plugin engine and Border Sync as I want to get started on their dev - note i wont always be able to work on them though

**Comment by themirrazz** [comment]
why is this a thing
![image](https://user-images.githubusercontent.com/89490659/219135191-386d7052-65cd-4fb8-b113-15b5cf79a2d6.png)

**Comment by kelbazz** [comment]
> hi uh i have an alt as for discord not yet but i can email you from my new email btw I was wondering if it would be okay if I created new repo(s) on Onofficiel for Border 1.11,Desktop, Android, and iOS, and for BRX plugin engine and Border Sync as I want to get started on their dev - note i wont always be able to work on them though

Yeah of course you can and don't force yourself to work on them if you don't have time lol.

**Comment by kelbazz** [comment]
Woops

**Comment by themirrazz** [comment]
lol. Btw I think Border should have localization (specifically different languages)

**Comment by kelbazz** [comment]
Yup. Also do you have another account for discord or smth cause kinda hard to talk there XD

**Comment by kelbazz** [comment]
And also for Border: What if we use an alternative to github ver managment like https://codeberg.org

**Comment by kelbazz** [comment]
I think this could be great

**Comment by themirrazz** [comment]
hi

**Comment by kelbazz** [comment]
Hi

**Comment by kelbazz** [comment]
Arg ! Talking via github is horrible lol

**Comment by kelbazz** [comment]
Do you have a gmail to talk on google chat

**Comment by themirrazz** [comment]
no i dont lol im at school rn

**Comment by themirrazz** [comment]
and im grounded

**Comment by themirrazz** [comment]
and my gmail account got blocked for security reasons

**Comment by themirrazz** [comment]
lol

**Comment by kelbazz** [comment]
Ooof !

**Comment by themirrazz** [comment]
lol

**Comment by themirrazz** [comment]
btw for editing repos you can use vscode replace github.com with github.dev

**Comment by kelbazz** [comment]
*This Hurts*

**Comment by kelbazz** [comment]
yep i know

**Comment by kelbazz** [comment]
or type `.` on the repo page

**Comment by themirrazz** [comment]
wait what you can type `.` on the repo page

**Comment by kelbazz** [comment]
yup or `ctrl + k` and `github.dev`

**Comment by themirrazz** [comment]
lol

**Comment by kelbazz** [comment]
Unrelated but i'll maybe use lapce as my editor now

**Comment by kelbazz** [comment]
https://lapce.dev/

**Comment by themirrazz** [comment]
lol

**Comment by kelbazz** [comment]
Why are you grounded btw ?

**Comment by kelbazz** [comment]
if you wanna say

**Comment by themirrazz** [comment]
I don't really want to say it here where my dad and his friends who are police officers that he threatened to get involved can easily just find it and harass me about talking bad about him

**Comment by themirrazz** [comment]
well not keeps

**Comment by kelbazz** [comment]
oh... ok

**Comment by themirrazz** [comment]
he's just... over-reacting.

**Comment by kelbazz** [comment]
:(

**Comment by themirrazz** [comment]
im grounded for a year... minimum

**Comment by themirrazz** [comment]
thats if I get _straight As_ for one year lmao

**Comment by kelbazz** [comment]
wtf !

**Comment by kelbazz** [comment]
Your parents are so strict !!

**Comment by themirrazz** [comment]
* my dad

**Comment by themirrazz** [comment]
oops lol

**Comment by kelbazz** [comment]
The worst for me is 1 week

**Comment by themirrazz** [comment]
not meant to be a bullet

**Comment by themirrazz** [comment]
theres also lot more to the story

**Comment by kelbazz** [comment]
:(

**Comment by themirrazz** [comment]
but anyways

**Comment by themirrazz** [comment]
lol

**Comment by themirrazz** [comment]
![image](https://user-images.githubusercontent.com/89490659/219796248-9aafb3cd-1943-4b26-b752-f1bfe4bc9001.png)
what

**Comment by themirrazz** [comment]
edit: there was an image but it showed where i go to school whoops

**Comment by themirrazz** [comment]
oooh
![image](https://user-images.githubusercontent.com/89490659/219796697-6dad4fdd-528e-4c44-9b9f-0f2441548348.png)
so thats why bc my school blocks win96 lol

**Comment by kelbazz** [comment]
yeah made some kind of w96 previewer for the app

**Comment by kelbazz** [comment]
Ohhh !!!

**Comment by kelbazz** [comment]
PERFECTION !!!

**Comment by kelbazz** [comment]
https://github.com/mozilla/qbrt

**Comment by themirrazz** [comment]
NO WAY

**Comment by kelbazz** [comment]
yes :,)

**Comment by kelbazz** [comment]
So we can build border in js without v8

**Comment by kelbazz** [comment]
i'm happy

**Comment by themirrazz** [comment]
FINALLY

**Comment by themirrazz** [comment]
its open source to

**Comment by themirrazz** [comment]
_\*ports to android\*_

**Comment by themirrazz** [comment]
we will modify it

**Comment by themirrazz** [comment]
I want it to run on windows XP

**Comment by themirrazz** [comment]
and Mac OS 9

**Comment by kelbazz** [comment]
:)))

**Comment by kelbazz** [comment]
can you code rn ?

**Comment by themirrazz** [comment]
no

**Comment by kelbazz** [comment]
If yes lets go start with this bad boy

**Comment by kelbazz** [comment]
oh

**Comment by kelbazz** [comment]
too bad

**Comment by kelbazz** [comment]
you're on mobile ?

**Comment by themirrazz** [comment]
no im grounded remember lol im at school

**Comment by kelbazz** [comment]
oh yeah

**Comment by kelbazz** [comment]
MST vs UTC+1

**Comment by kelbazz** [comment]
lol forgot

**Comment by kelbazz** [comment]
11:30pm

**Comment by kelbazz** [comment]
for me

**Comment by themirrazz** [comment]
lol

**Comment by kelbazz** [comment]
but can you access https://codeberg.org just to create your account

**Comment by kelbazz** [comment]
so you can reserve "themirrazz" as username

**Comment by themirrazz** [comment]
sure

**Comment by themirrazz** [comment]
edit 
![image](https://user-images.githubusercontent.com/89490659/219808436-1f020a8d-8dd4-4596-aedf-a0f1fa1bdb75.png)

**Comment by themirrazz** [comment]
oh wait i already have an account lmao

**Comment by kelbazz** [comment]
I follow you !

**Comment by themirrazz** [comment]
lol

**Comment by kelbazz** [comment]
Honestly this opensource github is incredible

**Comment by kelbazz** [comment]
even has pages

**Comment by kelbazz** [comment]
like https://kelbaz.codeberg.page/

**Comment by themirrazz** [comment]
btw the minesweeper link broke :(

**Comment by themirrazz** [comment]
oh nvm lol

**Comment by themirrazz** [comment]
me after 2 clicks lol
![image](https://user-images.githubusercontent.com/89490659/219811058-2c170b55-2156-4095-977d-0efb87c74d60.png)

**Comment by themirrazz** [comment]
i cannot write localizations rn there are too many

**Comment by themirrazz** [comment]
tell me if these are good ideas for border
* RSS feed reader
* Border Sync
* allow installing blugins from any site (with warnings, ofc)
* P3 connections (eg the RTOP protocol used by SuperTerm)
* another feedback thing lol

**Comment by themirrazz** [comment]
okay maybe not P3

**Comment by kelbazz** [comment]
Honestly yeah that's cool and doable

**Comment by kelbazz** [comment]
But maybe not p3 xd

**Comment by kelbazz** [comment]
I think tomorrow i'll start border desktop dev with qbrt

**Comment by stretch07** [comment]
@themirrazz HELLO I READ THE THREAD
I hope you get ungrounded soon
also I speak fluent spanish at home i can help with localizations if you'd like
let me know if there is a more conveniant/secure way to contact you

**Comment by themirrazz** [comment]
idk if any way is secure my dad has friends that are police in cybersecurity and the tech department is monitoring my Chromebook - apparently they can get the internet usage of things that happen when I'm not even signed in???

**Comment by themirrazz** [comment]
> I think tomorrow i'll start border desktop dev with qbrt

k

**Comment by themirrazz** [comment]
> @themirrazz HELLO I READ THE THREAD I hope you get ungrounded soon also I speak fluent spanish at home i can help with localizations if you'd like let me know if there is a more conveniant/secure way to contact you

also im not supposed to be chatting online lol

**Comment by themirrazz** [comment]
> But maybe not p3 xd

lol

**Comment by stretch07** [comment]
Alright i hope you can get through this sooner rather than later💫

**Comment by themirrazz** [comment]
> Alright i hope you can get through this sooner rather than later💫

most likely not lmao
